packages/compile-solidity/test/compileWithPragmaAnalysis.js

```diff
@@ -149,6 +144,25 @@ describe("compileWithPragmaAnalysis", function () {
   });
 
   describe("when there is a semver expression error", function () {
+    it("throws an error when a 'direct source' lacks a pragma expression", async function () {
```

Comment:
Question: Do we currently have a test that it does *not* error when an *imported* source lacks a pragma expression?  That might also be good to add.